bugbear.py

```diff
@@ -641,6 +643,19 @@ def is_abstract_decorator(expr):
 
         self.errors.append(B024(node.lineno, node.col_offset, vars=(node.name,)))
 
+    def check_for_b026(self, call: ast.Call):
+        try:
```

Comment:
This guard is good in general, but especially needed if we go with the suggestion above to avoid catching the empty keywords later on.

```suggestion
        if not call.keywords:
            return
        try:
```